telegram_bot/handlers/upload.py

```
@@ -0,0 +1,628 @@
+
+    await state.clear()
+    progress_message = await callback.message.answer("Обрабатывается... ⏳\nСтатус: pending\nПрогресс: 0%")
+    asyncio.create_task(
+        _monitor_lecture_progress(
+            callback.bot,
+            chat_id=progress_message.chat.id,
+            message_id=progress_message.message_id,
+            telegram_id=callback.from_user.id,
+            lecture_id=lecture_id,
+        )
+    )
```

Ruff (0.15.5)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)